# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/shadow-dom/declarative/declarative-shadow-dom-repeats.html.ini
# Commit: 1954ce9669b8
# Full Hash: 1954ce9669b86b61c418c8286fbe0754ad2419cd
# Author: Adam Vandolder <avandolder@mozilla.com>
# Date: 2024-05-15 20:24:18
# Regressor Bug: 1890428
# File Overlap Count: 1
# Description:
#   Bug 1890428 - Add serialization support for Declarative Shadow DOM. r=webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D206955
# ==============================================================================

diff -r 3e45e3d15eed -r 1954ce9669b8 testing/web-platform/meta/shadow-dom/declarative/declarative-shadow-dom-repeats.html.ini
--- a/testing/web-platform/meta/shadow-dom/declarative/declarative-shadow-dom-repeats.html.ini	Wed May 15 03:01:18 2024 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,3 +0,0 @@
-[declarative-shadow-dom-repeats.html]
-  [Calling attachShadow() on declarative shadow root must match all parameters]
-    expected: FAIL